rebasehelper/spec_content.py

```diff
@@ -87,6 +87,12 @@ def __str__(self):
                 content.append(line + '\n')
         return ''.join(content)
 
+    def __getitem__(self, index):
+        try:
+            return self.sections[index]
+        except IndexError:
+            raise KeyError
```

Comment:
What's the reasoning behind changing `IndexError` to `KeyError`?

Reply:
I thought it was a convention to raise `KeyError` if the used key can't be found. Or at least that's what I have seen most of the time

Reply:
What key? It's index. You even named it `index`.